com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -628,7 +627,7 @@ internal static void HandleNetworkVariableDeltas(List<INetworkVariable> networkV
                         }
                     }
 
-                    if (networkManager.IsServer && !networkVariableList[i].CanClientWrite(clientId))
+                    if (networkManager.IsServer)// ?? && !networkVariableList[i].CanClientWrite(clientId))
```

Comment:
Is this even needed anymore? because essentially its checking to see if the client can actually write to this value and if not log it... I am not sure this would be possible anymore correct?